### PR TITLE
map: remove ineffectual code

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

### CODE REVIEW

File: components/AvalancheForecastZoneMap.tsx [hunk 1]
```diff
@@ -76,7 +76,6 @@ export const AvalancheForecastZoneMap: React.FunctionComponent<MapProps> = ({cen
   const avalancheCenterMapRegionBounds: RegionBounds = zones
     ? zones.reduce((accumulator, currentValue) => updateBoundsToContain(accumulator, toLatLngList(currentValue.geometry)), defaultAvalancheCenterMapRegionBounds)
```
Comment (floatplane) [inline components/AvalancheForecastZoneMap.tsx:77]: using `defaultAvalancheCenterMapRegionBounds` as the accumulator will modify that constant in place, is that the right thing to do?

maybe `defaultAvalancheCenterMapRegionBounds` should instead be a function that returns an empty bounds object every time it's called.

**Review by floatplane — CHANGES_REQUESTED** [review]
ptal @stevekuznetsov

File: components/AvalancheForecastZoneMap.tsx (hunk 1 above)
Comment (floatplane) [reply components/AvalancheForecastZoneMap.tsx:77]: or do I have this wrong, and `updateBoundsToContain` doesn't modify its argument in place? if that's the case then ignore my feedback!

Comment (stevekuznetsov) [reply components/AvalancheForecastZoneMap.tsx:77]: Yeah it is written specifically not to mutate the input

Comment (stevekuznetsov) [reply components/AvalancheForecastZoneMap.tsx:77]: (and there's a test for that) :)

**Pull request merged**